### PR TITLE
Interface streaming bluetooth states with Angular Change detection.

### PR DESCRIPTION
**Pull request by cowboyd**
## Purpose

When bluetooth scans and connections and disconnections were happening, they were not updating the UI until some other event was triggering an update. As a result, the direct connect page would be blank until all the scan was completely finished, at which point all of the devices would pop in at one time.

The reason is that the callbacks for discovered peripherals are being invoked directly from the underlying bluetooth system and so were dropping the Angular context from which those callbacks were registered.

## Approach
This change makes sure that every invocation of the observable state is invoked within a call to `NgZone#run()` that way, no matter what component or template is subscribed to the observable, its changes will be received within the context of an Angular Zone. and can be checked for changes with the previous state.

### Before

All peripherals seem to appear at the same time

### After

Peripherals appear as they are discovered.

### Notes

- There was an errant dependency on `nativescript-uuid` which has been removed.
- This adds a 500ms timeout to the bluetooth scan start in order to make sure that it begins well after the nativescript-bluetooth subsystem is initialized.

### CODE REVIEW

File: src/bluetooth/store.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function BluetoothStore(fn: (state: Model) => any): Model {
     .match({
       guard: bluetooth => bluetooth.scan.isActivating,
       setup: () => {
-        ble.startScanning();
+        setTimeout(() => ble.startScanning(), 500);
```
Comment (cowboyd) [inline src/bluetooth/store.ts]: It's worth noting that the state transition stays exactly the same, and only the _effect_ of the the state transition changes. The in memory model is completely decoupled.... pretty powerful stuff.

**Comment by taras** [comment]
@cowboyd why are the tests failing?

**Comment by taras** [comment]
@cowboyd is this now superseded by #83?

**Comment by cowboyd** [comment]
This got rolled into #83

**Pull request closed without merging**